nixos/modules/services/networking/bind.nix

```diff
@@ -8,6 +8,35 @@ let
 
   bindUser = "named";
 
+  bindZoneOptions = {
+    name = mkOption {
+      type = types.str;
+      description = "Name of the zone.";
```

Comment:
please remove trailing dots

Reply:
Option descriptions are supposed to end in a period. They aren't a short description and can consist of multiple sentences.